src/Input.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  children: PropTypes.node,
+  type: PropTypes.string,
+  size: PropTypes.string,
+  state: PropTypes.string,
+  tag: PropTypes.string,
+  'static': PropTypes.bool,
+  className: PropTypes.string,
+};
+
+const defaultProps = {
+  tag: 'p',
```

Comment:
I wasn't sure why this would ever happen but then I remembered the static text class. 👍